tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino

```diff
@@ -48,9 +48,10 @@ const char kSolaxError[] PROGMEM =
 
 #ifdef SOLAXX1_READCONFIG
 const char kSolaxSafetyType[] PROGMEM =
-  "VDE0126|ARN4105|AS4777_AU|G98/1|C10/11|OVE/ONORME8001|EN50438_NL|EN50438_DK|CEB|CEI021|NRS097_2_1|"
-  "VDE0126_Gr_Is|UTE_C15_712|IEC61727|G99/1|VDE0126_Gr_Co|France_VFR2014|C15_712_is_50|C15_712_is_60|"
-  "AS4777_NZ|RD1699|Chile|EN50438_Ireland|Philippines|Czech_PPDS|Czech_50438";
+  "VDE 0126|VDE-AR-N 4105|AS 4777|G98|C10/11|ÖVE/ÖNORM E 8001|EN 50438 NL|EN 50438 DK|CEB|CEI021|NRS 097-2-1|VDE 0126 Greece/Iceland|"
```

Comment:
pls remove all this and if you feel you need to report something just display the number and let users search the solaxx documentation for it's meaning. Saves 1k code space to be used for important functions.

Reply:
In general I agree with you. For saving code space, I introduced the compiler option `#define SOLAXX1_READCONFIG `. Disabling this, will saves more than 3k space. So users, who need the space, must not use this unimportant function.

Reply:
Please disable this define by default. Thx.

Reply:
OK. Done.